scivision/io/reader.py

```diff
@@ -11,8 +11,10 @@
 from ..koala import koala
 from .installer import install_package
 from .wrapper import PretrainedModel
```

Comment:
```suggestion
from .wrapper import PretrainedModel, Datasource
```
Is this suggestion ok?